spi/common/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/security/VaultCertificateResolver.java

```diff
@@ -45,7 +45,7 @@ public VaultCertificateResolver(@NotNull Vault vault) {
         }
 
         try {
-            var encoded = certificateRepresentation.replace(HEADER, "").replaceAll(System.lineSeparator(), "").replace(FOOTER, "");
+            var encoded = certificateRepresentation.replace(HEADER, "").replaceAll(System.lineSeparator(), "").replace(FOOTER, "").replace("\n", "");
```

Comment:
Will this need to be merged back into the EDC code proper, or could we fix this also by removing newlines from the certs stored in HashiCorp Vault?

Reply:
this can be fixed on our side by removing the newlines